src/main/java/org/testeditor/web/xtext/index/serialization/EObjectDescriptionSerializer.xtend

```diff
@@ -13,6 +13,9 @@ import static org.testeditor.web.xtext.index.serialization.EObjectDescriptionSer
 import static org.testeditor.web.xtext.index.serialization.EObjectDescriptionSerialization.QUALIFIED_NAME__FIELD_NAME
 import static org.testeditor.web.xtext.index.serialization.EObjectDescriptionSerialization.URI__FIELD_NAME
 
+/**
+ * Serializer for EObjectDescription to Json
+ */
```

Comment:
Repeating class name again

Reply:
yeahh, I know, still I added the 'to Json' which does not make that shiny and good, but might work as an excuse here, doesn't it?